docs/howto/configure/auth-management.md

```diff
@@ -53,4 +55,107 @@ So we want to manage authentication by:
+   - The authorisation callback URL is the homepage url appended with `/hub/oauth_callback`
+   - Once you have created the OAuth app, make a new of the client ID, generate a client secret and then hold on to these values for a future step
+
+2. **Transfer the OAuth App to the `2i2c-org` GitHub account.**
+   By default, OAuth apps are created under your GitHub account.
+   However, it will be a point of failure if a hub's app is only accessible by a single member of the 2i2c Engineering team.
+   Therefore, please ensure you transfer it to the `2i2c-org` GitHub org by following [GitHub's documentation](https://docs.github.com/en/developers/apps/managing-oauth-apps/transferring-ownership-of-an-oauth-app).
+
+3. **Create or update the appropriate secret config file under `secrets/config/hubs/*.cluster.yaml`.**
+   You should add the following config to this file, pasting in the client ID and secret you generated in step 1.
+
+    ```yaml
+    hubs:
+    - name: HUB_NAME
+      config:
+        jupyterhub:
+          hub:
+            config:
+              GitHubOAuthenticator:
+                client_id: CLIENT_ID
+                client_secret: CLIENT_SECRET
+    ```
+
+    ```{note}
+    Add the `basehub` key between `config` and `jupyterhub` for `daskhub` deployments.
+    ```
+
+    ```{admonition}
+    Make sure this is encrypted with `sops` before committing it to the repository!
+
+    `sops -i -e secrets/config/hub/*.cluster.yaml`
+    ```
+
+4. **Edit the non-secret config under `config/hubs`.**
+   You should make sure the matching hub config takes one of the following forms.
+
+   To authenticate against a GitHub organisation:
+
+    ```yaml
+    hubs:
+    - name: HUB_NAME
+      auth0:
+        enabled: false
+        connection: ""  # This key is still required by the schema
+      ... # Other config
+      config:
+        jupyterhub:
+          hub:
+            config:
+              JupyterHub:
+                authenticator_class: github
+              GitHubOAuthenticator:
+                oauth_callback_url: https://{{ HUB_DOMAIN }}/hub/oauth_callback
+                allowed_organizations:
+                  - 2i2c-org
+                  - ORG_NAME
+                scope:
+                  - read:user
+            extraConfig:
+              06-custom-authenticator: ""  # Required to override our overrides...
+    ```
+
+   To authenticate against a GitHub Team:
+
+    ```yaml
+    hubs:
+    - name: HUB_NAME
+      auth0:
+        enabled: false
+        connection: ""  # This key is still required by the schema
+      ... # Other config
+      config:
+        jupyterhub:
+          hub:
+            config:
+              JupyterHub:
+                authenticator_class: github
+              GitHubOAuthenticator:
+                oauth_callback_url: https://{{ HUB_DOMAIN }}/hub/oauth_callback
+                allowed_organizations:
+                  - 2i2c-org:tech-team
+                  - ORG_NAME:TEAM_NAME
```

Comment:
I included the `2i2c-org:tech-team` here since, when I tried this on the Pangeo staging deployment, I was denied access until I added myself to that team. I wasn't sure which way round we'd need like to manage this going forward, either always have the `2i2c-org:tech-team` listed in `allowed_organizations` or always add ourselves to `ORG_NAME:TEAM_NAME`.

Same discussion applies to the "just org" example above this.

Reply:
it sounds like this one got resolved? is that right?

Reply:
I don't think we've had the discussion yet? I'd like the team to agree on a process to either always including `2i2c-org:tech-team` in the config, or being added to the community's org/team

Reply:
I would go with the first option, I think...

Reply:
@sgibson91 ah sorry I had misunderstood - my feeling is that we should go with option 1 because that's an action we have more control over, and I think it will be more reliable to rely on ourselves rather than other communities to take action. Does that make sense?

Reply:
Ok great, in which case there's no action as the documentation is already reflecting option 1 :)